homeassistant/components/tts/amazon_polly.py

```diff
@@ -148,6 +146,10 @@ def supported_options(self):
 
     def get_tts_audio(self, message, language=None, options=None):
         """Request TTS file from Polly."""
+        if language not in self.all_languages:
```

Comment:
I thought you told me to add this in [this comment](https://github.com/home-assistant/home-assistant/pull/5169#discussion_r97724929)...